daemon/daemon.cc

```diff
@@ -711,6 +734,8 @@ void tr_daemon::reconfigure()
 
         tr_sessionSet(my_session_, load_settings(configDir));
         tr_sessionReloadBlocklists(my_session_);
+
+        sd_notify(0, "STATUS=Reload complete.\nREADY=1\n");
```

Comment:
Why this line is not guarded with ifdef also?

Reply:
Because `sd_notify` and `sd_notifyf` is no-op when `WITH_SYSTEMD` is not defined.

https://github.com/transmission/transmission/blob/f80ae85fd07442bda244bf4e509ea7e5060f822c/daemon/daemon.cc#L59-L63